### ISSUE TITLE
feat req: Add Uptime Kuma Support

### ISSUE THREAD

**Issue by Luciousdev** [Luciousdev/server-dashboard-API#1]
**Checklist**

- [x] I have checked that this issue/feature request does not already exist.

**Is your feature request related to a problem? Please describe.**
No it is not 

**Describe the solution you'd like**
We'd like to be able to retrieve the data from uptime kuma and to be able to use it in this API.

**Describe alternatives you've considered**
NOT APPLICABLE

## Tasklist
- [x] Get monitors with their status and ping.
- [x] Login in using username and password

**Comment by Luciousdev** [comment]
Backend Tests PASSED

**Comment by Luciousdev** [comment]
Currently waiting on frontend to approve changes. And create the frontend for it. 

Also additional features for uptime kuma can later be requested with a new issue.

**Comment by Luciousdev** [comment]
Frontend does not need to approve changes anymore, if anything needs to change a issue can be opened later.
